### PR TITLE
built term and conditions screen

### PR DESCRIPTION
**Pull request by ayeshaalyas**
##jira
[feature/TS-8-build-terms-conditions-screen](https://github.com/Shop-G/ShopGo/compare/develop...feature/TS-8-build-terms-conditions-screen?expand=1)

Description:
-> build the UI for terms and condition screen
-> click on read more button will extend label text and and also change title and vice versa

How to test it?
-> click on read more button will extend the view and title of label to read less 

Screen shoots and Gifs:

Screen Shot 2022-09-21 at 7 26 51 AM

### CODE REVIEW

File: TermsAndConditions/TermsAndConditionVC.swift [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+import UIKit
+
+class TermsAndConditionVC: UIViewController {
+    
+    @IBOutlet weak var textLablel: UILabel!
+    @IBOutlet weak var readMoreLablel: UILabel!
+    @IBOutlet weak var arrowImg: UIImageView!
+    
+    var isReadMoreTapped = false
```
Comment (haroonmaqsoodch) [inline TermsAndConditions/TermsAndConditionVC.swift:16]: move var above the the outlets

File: TermsAndConditions/TermsAndConditionVC.swift [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+
```
Comment (haroonmaqsoodch) [inline TermsAndConditions/TermsAndConditionVC.swift:22]: remove extra space from viewDidLoad()

File: TermsAndConditions/TermsAndConditionVC.swift [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    }
+    
+    @IBAction func readMoreAction(_ sender: UIButton) {
+        if isReadMoreTapped {
+            readMoreLablel.text = "Read more"
+            textLablel.numberOfLines = 8
+            arrowImg.image = UIImage(systemName: "chevron.down")
+            isReadMoreTapped = false
+        }else {
```
Comment (haroonmaqsoodch) [inline TermsAndConditions/TermsAndConditionVC.swift:31]: one space before else

File: TermsAndConditions/TermsAndConditionVC.swift [hunk 4]
```diff
@@ -0,0 +1,39 @@
+//
```
Comment (haroonmaqsoodch) [inline TermsAndConditions/TermsAndConditionVC.swift:1]: move your swift file and storyboard file out of the folder with other ViewController and main storyboard. Delete the group after you move them

**Review by haroonmaqsoodch — COMMENTED** [review]
There is no show less button in the design, the chevron should move to the right side in order to make the text "less" please fix this change as per design

**Comment by ayeshaalyas** [comment]
okay

**Review by basitalich — CHANGES_REQUESTED** [review]

File: TermsAndConditions/TermsAndConditionVC.swift [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+
+    }
+    
+    @IBAction func readMoreAction(_ sender: UIButton) {
+        if isReadMoreTapped {
```
Comment (basitalich) [inline TermsAndConditions/TermsAndConditionVC.swift:26]: fix indentation and spacing

**Review by basitalich — CHANGES_REQUESTED** [review]

File: TermsAndConditions/TermsAndConditionVC.swift [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+//  ShopGo
+//
+//  Created by Ayesha Alyas on 9/20/22.
+//
+
+import UIKit
+
+class TermsAndConditionVC: UIViewController {
+    
+    @IBOutlet weak var textLablel: UILabel!
```
[the remark this answers is not in the document]
Comment (ayeshaalyas) [reply TermsAndConditions/TermsAndConditionVC.swift]: i am using a label inside a view to display text

File: TermsAndConditions/TermsAndConditionVC.swift [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+    @IBOutlet weak var readMoreLablel: UILabel!
+    @IBOutlet weak var arrowImg: UIImageView!
+    
+    var isReadMoreTapped = false
+    
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
```
[the remark this answers is not in the document]
Comment (ayeshaalyas) [reply TermsAndConditions/TermsAndConditionVC.swift]: i removed spaces

File: TermsAndConditions/TermsAndConditionVC.swift (hunk 5 above)
Comment (ayeshaalyas) [reply TermsAndConditions/TermsAndConditionVC.swift]: i did

**Review by basitalich — APPROVED** [review]

**Pull request merged**